src/typings/project.tsx

```diff
@@ -2,3 +2,6 @@ export interface ProjectElement {
     id: number;
     title: string;
 }
+
+export interface GeoOptions {
+}
```

Comment:
Does this have no property?

Reply:
At this moment the properties are unknown so I left them

Reply:
Let's add a comment that the properties are unknown.